packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/NewCollaborator.spec.js

```diff
@@ -0,0 +1,572 @@
+  shareButton: '#files-collaborators-collaborator-save-new-share-button',
+  savingInProgress: 'oc-spinner-stub'
+}
+
+describe('Edit Collaborator', () => {
+  describe('user/group input', () => {
+    it('should display the input label', () => {
+      const wrapper = getShallowMountedWrapper({
+        user: 'user0'
+      })
+
+      const userInviteLabel = wrapper.find(selectors.userInviteLabel)
+
+      expect(userInviteLabel.exists()).toBe(true)
+      expect(userInviteLabel.text()).toBe('Invite')
+    })
+
+    it('should display the user invite input hint', () => {
+      const wrapper = getShallowMountedWrapper({
+        user: 'user0'
+      })
+
+      const userInviteHint = wrapper.find(selectors.userInviteHint)
+
+      expect(userInviteHint.exists()).toBe(true)
+      expect(userInviteHint.text()).toBe('Add new person by name, email or federation IDs')
+    })
+
+    it('show display the user select input', () => {
+      const wrapper = getMountedWrapper({
+        user: 'user0'
+      })
+      const userSelectInput = wrapper.find(selectors.userInviteSelect)
+      expect(userSelectInput.exists()).toBe(true)
+    })
+
+    it('should call Fetch sharees request with correct query from the input', async () => {
+      const autoCompleteTrigger = jest.spyOn(NewCollaborator.methods, 'fetchRecipients')
+
+      const wrapper = getMountedWrapper({
+        user: 'user0'
+      })
+
+      localVue.prototype.$client.shares.getRecipients = () => ({
+        exact: {
+          users: [],
+          groups: [],
+          remotes: []
+        },
+        users: [],
+        groups: [],
+        remotes: []
+      })
+
+      const userSelectInput = wrapper.find(selectors.userInviteInput)
+
+      await userSelectInput.setValue('use')
+
+      // the api request for fetching the user is triggred few seconds after the input is set
+      await new Promise(resolve => {
+        setTimeout(() => {
+          expect(autoCompleteTrigger).toHaveBeenCalledTimes(1)
+          expect(autoCompleteTrigger).toHaveBeenCalledWith('use')
+          resolve()
+        }, 3000)
+      })
+    })
+
+    it('should show no results when no users or groups are returned', async () => {
+      const wrapper = getMountedWrapper({
+        user: 'user0'
+      })
+      localVue.prototype.$client.shares.getRecipients = () => ({
+        exact: {
+          users: [],
+          groups: [],
+          remotes: []
+        },
+        users: [],
+        groups: [],
+        remotes: []
+      })
+
+      const userSelectInput = wrapper.find(selectors.userInviteInput)
+      await userSelectInput.setValue('use')
+
+      await new Promise(resolve => {
+        setTimeout(() => {
+          const select = wrapper.findComponent(VueSelect)
+          expect(select.props('options')).toMatchObject([])
+          resolve()
+        }, 3000)
+      })
+    })
+
+    it('Users should be listed in the autocomplete dropdown', async () => {
```

Comment:
starting scenario title with `should` would be nicer and uniform 
```suggestion
    it('should display users list in the autocomplete dropdown', async () => {
```